app/js/daemonManager.js

```diff
@@ -72,17 +72,21 @@ function DaemonManager() {
 		// duration that it takes to load up the blockchain.
 		apiCall("/daemon/updates/check", function(err, update) {
 			if (err) {
+				console.log(err, update)
 				self.Running = false;
 				// Check again later
-				setTimeout(updatePrompt, 1000);
+				setTimeout(updatePrompt, 5000);
```

Comment:
didn't realize it's an update check.

5 seconds is probably more than needed too. Even 10 minutes should be fine as far as software updates go.